src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -353,27 +353,29 @@ void preferences_dialog::post_build(window& window)
 	//
 
 	/* FULLSCREEN TOGGLE */
-	toggle_button& toggle_fullscreen =
-			find_widget<toggle_button>(&window, "fullscreen", false);
-
-	toggle_fullscreen.set_value(fullscreen());
-
-	// We bind a special callback function, so setup_single_toggle() is not used
-	connect_signal_mouse_left_click(toggle_fullscreen, std::bind(
-			&preferences_dialog::fullscreen_toggle_callback,
-			this, std::ref(window)));
+	toggle_button* toggle_fullscreen =
+			find_widget<toggle_button>(&window, "fullscreen", false, false);
+	
+	if (toggle_fullscreen != nullptr) {
```

Comment:
Personally I'd prefer the `if(toggle_button* toggle_fullscreen = find_widget ...)` idiom for this sort of thing.